cli/Cargo.toml

```diff
@@ -47,7 +47,7 @@ ring = "0.16.15"
 rustyline = { version = "6.2", default-features = false }
 serde = { version = "1.0.112", features = ["derive"] }
 serde_derive = "1.0.112"
-serde_json = { version = "1.0.55", features = [ "preserve_order" ] }
```

Comment:
I think `preserve_order` was switched on on purpose, it's not active by default.

```
# Make serde_json::Map use a representation which maintains insertion order.
# This allows data to be read into a Value and written back to a JSON string
# while preserving the order of map keys in the input.
```
https://github.com/serde-rs/json/blob/master/Cargo.toml